rna_cd/cli.py

```diff
@@ -0,0 +1,186 @@
+              help="Path where model will be stored.")
+def train_cli(chunksize: int, contig: str, model_out: Path,
+              positives_dir: Optional[List[Path]] = None,
+              negatives_dir: Optional[List[Path]] = None,
+              positives_list: Optional[List[Path]] = None,
+              negatives_list: Optional[List[Path]] = None,
+              cross_validations: int = 3,
+              verbosity: int = 1, cores: int = 1,
+              plot_out: Optional[str] = None):
+
+    if positives_dir is None and positives_list is None:
+        raise ValueError("Must set either --positives-dir or --positives-list")
+
+    if negatives_dir is None and negatives_list is None:
+        raise ValueError("Must set either --negatives-dir or --negatives-list")
+
+    positives = positives_dir if positives_dir is not None else positives_list
+    negatives = negatives_dir if negatives_dir is not None else negatives_list
+
+    model = train_svm_model(positives, negatives, chunksize=chunksize,
+                            contig=contig, cross_validations=cross_validations,
+                            verbosity=verbosity, cores=cores,
+                            plot_out=plot_out)
+
+    save_sklearn_object_to_disk(model, Path(model_out))
+
+
+@click.command()
+@click.option("--chunksize", type=click.INT, default=100,
+              help="Chunksize in bases. Default = 100")
+@click.option("-c", "--contig", type=click.STRING, default="chrM",
+              help="Name of mitochrondrial contig in your BAM files. "
+                   "Default = chrM")
+@click.option("-j", "--cores", type=click.INT, default=1,
+              help="Number of cores to use for processing of BAM files. "
+                   "Default = 1")
+@click.option("-d", "--directory",
+              type=click.Path(exists=True, readable=True,
+                              dir_okay=True, file_okay=False),
+              callback=directory_callback,
+              help="Path to directory with BAM files to be tested. "
+                   "Mutually exclusive with --list-items")
+@click.option("-l", "--list-items",
+              type=click.Path(exists=True, readable=True,
+                              file_okay=True, dir_okay=False),
+              callback=list_callback,
+              help="Path to file containing list of paths to BAM files to be "
+                   "tested. Mutually exclusive with --directory")
+@click.option("-m", "--model",
+              type=click.Path(exists=True, readable=True,
+                              file_okay=True, dir_okay=False),
+              callback=path_callback, required=True,
+              help="Path to model."
+              )
+@click.option("-o", "--output", type=click.Path(writable=True),
+              required=True, callback=path_callback,
+              help="Path to output file containing classifications.")
+def classify_cli(chunksize: int, contig: str, cores: int,
+                 directory: Optional[List[Path]],
```

Comment:
Because the default is `None` if the user doesn't use `--directory`. Setting a default `[]` would get us in the hairy mutable default argument mess.